api/src/main/java/com/datastrato/gravitino/authorization/AuthorizationRole.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.authorization;
+
+/** Authorization Role interface. */
+public interface AuthorizationRole {
+  /**
+   * Create a role.
+   *
+   * @param name The role's name.
+   * @return The created role object.
+   * @throws UnsupportedOperationException If the authorization does not support to create a role.
+   */
+  Role createRole(String name) throws UnsupportedOperationException;
```

Comment:
Do you need to return the `Role`? I think it is not possible to get a `Role` object from underlying system as we discussed before.

Reply:
Is `name` parameter alone enough to create a role?

Reply:
I reconstructed this `AuthorizationRole.java`